lib/src/executable/compile_stylesheet.dart

```diff
@@ -127,13 +127,20 @@ Future<void> compileStylesheet(ExecutableOptions options, StylesheetGraph graph,
 
   if (options.quiet || (!options.update && !options.watch)) return;
   var buffer = StringBuffer();
-  if (options.color) buffer.write('\u001b[32m');
 
   var sourceName = source == null ? 'stdin' : p.prettyUri(p.toUri(source));
   // `destination` is guaranteed to be non-null in update and watch mode.
   var destinationName = p.prettyUri(p.toUri(destination!));
+
+  var nowStr = DateTime.now().toString();
+  // Remove fractional seconds from printed timestamp
+  var timestamp = nowStr.substring(0, nowStr.length - 7);
+
+  buffer.write('$timestamp - ');
```

Comment:
Let's format this like `[$timestamp] $message`. Also, I do think the timestamp should be grey (you can ignore wojtekxtx's comments, I don't think they understand how terminal colors work).

Reply:
I really like the `[ ]` formatting suggestion – didn't really care for the extra hyphen in my version (or an extra colon for that matter). I checked the gray color with a translucent black terminal and it looks really slick... enough so that I may keep my background that color:
 
![image](https://user-images.githubusercontent.com/3347095/218639807-fc46ba9e-80e6-4c8c-97ff-a51e056310aa.png)

Reply:
@nex3 oh man, I do understand. Also I do understand much more than you tbh.